socorro/signature/__main__.py

```diff
@@ -157,7 +157,16 @@ def main(args):
     parser.add_argument(
         'crashids', metavar='crashid', nargs='*', help='crash id to generate signatures for'
     )
-    args = parser.parse_args()
+
+    # Fix the case where the user ran "python -m socorro.signature --help" so
+    # it prints a helpful prog.
+    if sys.argv and '__main__.py' in sys.argv[0]:
+        parser.prog = 'python -m socorro.signature'
```

Comment:
Huh, I did not know this was an attribute; only the kwarg is documented.

Reply:
Mmm... I suppose I could do the same thing I did in socorro-cmd and just replace `sys.argv[0]`. Given there's precedence for that (though it's a little icky), I'll switch it.